### PR TITLE
Make it possible to view site without logging in

### LINKED ISSUE #44: Be able to view site without logging in
Logging in should only be required when placing a review (or adding place?).

### PR DESCRIPTION
**Pull request by Kalior**
Fixes #44 

2019-05-12-214049_1892x895_scrot
Not sure about the design.  Pressing the `Logga in` button on the top right prompts login from google.

### CODE REVIEW

File: src/components/UserController.tsx [hunk 1]
```diff
@@ -9,13 +8,23 @@ import {
...
+  const { googleUser, user, authorize } = useGoogleAuth();
 
   const dispatch = useDispatch();
+  const [displayName, setDisplayName] = useState(user ? user.displayName : "");
 
-  const [displayName, setDisplayName] = useState(user.displayName);
+  if (!googleUser || !user) {
+    return (
+      <>
+        <h1>Authorise google to use this page.</h1>
```
Comment (lindskogen) [inline src/components/UserController.tsx]: Separate the two code paths inte components

File: src/components/place/CommentField.tsx [hunk 2]
```diff
@@ -14,19 +14,21 @@ interface Props {
...
-      </div>
-      <RatingContainer>
-        {review ? (
-          <CommentOnly review={review} />
-        ) : (
-          <NewCommentForm placeId={placeId} />
-        )}
-      </RatingContainer>
-    </CommentContainer>
+    user && (
```
Comment (lindskogen) [inline src/components/place/CommentField.tsx]: Early `return null` instead?

Or handle no user by rendering a default user icon and something like “Logga in för att kommentera”

File: src/App2.tsx [hunk 3]
```diff
@@ -19,17 +18,6 @@ declare global {
 }
 
 export const App2: React.FC = () => {
-  const { googleUser, user, authorize } = useGoogleAuth();
```
Comment (lindskogen) [inline src/App2.tsx]: With this change, we might not need App2 any more!

File: src/components/StatusBar.tsx [hunk 4]
```diff
@@ -63,6 +87,31 @@ const NavItem = styled(NavLink)`
...
+  width: 25%;
+
+  background-color: rgb(227, 227, 227);
+`;
+
+const Item = styled.div`
+  color: #fff;
+  text-decoration: none;
+  padding: 0.8rem 1.2rem;
+  width: 25%;
```
Comment (lindskogen) [inline src/components/StatusBar.tsx]: Extract shared css

File: src/components/place/CommentField.tsx (hunk 2 above)
Comment (Kalior) [reply src/components/place/CommentField.tsx]: I figured rendering "logga in för att kommentera" on every card would look dumb, but we may need some UX expertise here.

**Pull request merged**